pkg/operator/ceph/controller/spec.go

```diff
@@ -798,3 +800,12 @@ func ConfigureExternalMetricsEndpoint(ctx *clusterd.Context, monitoringSpec ceph
 func extractMgrIP(rawActiveAddr string) string {
 	return strings.Split(rawActiveAddr, ":")[0]
 }
+
+func GetContainerImagePullPolicy(containerImagePullPolicy string, opConfig OperatorConfig) *v1.PullPolicy {
+	imagePullPolicy := k8sutil.GetValue(opConfig.Parameters, "ROOK_CEPH_IMAGE_PULL_POLICY", containerImagePullPolicy)
```

Comment:
`GetContainerImagePullPolicy`  only checks for `ROOK_CEPH_IMAGE_PULL_POLICY`. But we have `ROOK_CSI_IMAGE_PULL_POLICY` in the config map as well.

Reply:
For CSI driver a new param has been introduced and set here https://github.com/rook/rook/pull/10966/files#diff-0a689560984ab97a96a578513a7ea7a64ec112156e02c199d447baf0a1378a74R271